### PR TITLE
Add user association to a rover

### PR DESCRIPTION
**Pull request by cabarnes**
Related to #19 in order to modify the user's rovers. Changes `owner` to be an actual user. There's no easy migration from a string to a foreign key, so the `Rover` objects will all need to be removed when the migrations are applied.

### CODE REVIEW

File: mission_control/serializers.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from rest_framework import serializers
 
 
-class RoverSerializer(serializers.HyperlinkedModelSerializer):
+class RoverSerializer(serializers.ModelSerializer):
```
Comment (aninternetof) [inline mission_control/serializers.py]: I'm reading about the differences between these two. The HyperlinkedModelSerializer does not lend itself to use with a foreign key?

**Comment by aninternetof** [comment]
I went to https://beta.rovercode.com/mission-control/rovers/ to delete the rovers before we merge this PR. But, I'm getting a `500` error.  Was this expected from previous changes? (I didn't think anything about the model changed in the previous PR).
I can always delete them within Django, but I'm surprised we didn't have a test for this DRF view.
If I get a minute over lunch, I'll SSH into beta.rovercode.com and try to see what's up.

File: mission_control/serializers.py (hunk 1 above)
Comment (cabarnes) [inline mission_control/serializers.py]: The issue I was running into was that the `User` model didn't have a serializer. If we eventually want to implement that we can put this back. It just didn't seem all that important.

**Comment by cabarnes** [comment]
I wouldn't have expected an issue. Everything is still running locally for me. The only change to the model recently was adding the pin assignment fields which was a migration.

**Comment by aninternetof** [comment]
Huh, interesting.

And actually, there shouldn't be any `rover` records, because on every view, they get deleted if they haven't check in recently.

**Comment by aninternetof** [comment]
I'll go ahead an merge this PR and see where we end up.

File: mission_control/serializers.py (hunk 1 above)
Comment (aninternetof) [inline mission_control/serializers.py]: Ah! That makes sense.

**Pull request merged**